tests/features/custom_page/custom_page.search_field.feature

```diff
@@ -0,0 +1,38 @@
+@api
+Feature: "Custom page" search.
+  In order to use the search
+  As a user of the website
+  I need to be able to view search results in the "Custom page" canonical page.
+
+  Background:
+    Given the following collection:
+      | title  | Code camp  |
+      | logo   | logo.png   |
+      | banner | banner.jpg |
+    And discussion content:
+      | title              | content                        |
+      | Discussion example | This is some dummy discussion. |
+    And "custom_page" content:
+      | title    | collection | body               |
+      | About us | Code camp  | This is an example |
+    # Non UATable step.
+    Given I commit the solr index
+
+    Scenario: View search results in the "Custom page" canonical page.
+      When I am an anonymous user
+      And I go to the homepage of the "Code camp" collection
+      And I click "About us"
+      Then I should see the heading "Discussion example"
```

Comment:
I don't see the heading "Discussion example" here. I have clicked on "About us" and obviously I see the heading "About us" now. This test is only green because the title of the "Discussion example" tile happens to be rendered with a H2 element which is also considered as a header.

The correct step would be:

> Then I should see the "Discussion example" tile in the related content section

The functionality is wrong, we cannot accept it like this. Why is the expected behaviour that when the user created an About Us page there are links to unrelated discussions being shown on the page? As a collection owner I want to have control over what appears on my custom pages. A simple custom page should not have any related content on it by default.

This test should actually be inverted. It should be like this:

```
Scenario: Related content does not show by default on a custom page
  Given I am an anonymous user
  When I go to the homepage of the "Code camp" collection
  And I click "About us"
  Then I SHOULD NOT SEE the "Discussion example" tile in the related content section
```